starport/pkg/gocmd/gocmd.go

```diff
@@ -72,10 +72,12 @@ func BuildAll(ctx context.Context, out, path string, flags []string, options ...
 }
 
 // InstallAll runs go install ./... on path with options.
-func InstallAll(ctx context.Context, path string, flags []string, options ...exec.Option) error {
+func InstallAll(ctx context.Context, binary, path string, flags []string, options ...exec.Option) error {
 	command := []string{
 		Name(),
-		CommandInstall,
+		CommandBuild,
+		FlagI,
+		FlagOut, binaryPath(binary),
```

Comment:
```suggestion
		FlagOut,
		binaryPath(binary),
```

Reply:
Not very important to me. I just find it unclean to have one value per line and several on the last line, didn't know it's a pattern

Reply:
I'm not sure how this line work with "./..." below.

"./..." means recursively install all the main programs. but `binaryPath(binary)` returns a single file name and not a directory name. these two seemed conflicting to me, can you please check it?

Reply:
@Pantani instead of changing the logic of InstallAll maybe you add a new Build func?

Reply:
@ilgooz `./...` woks because only have one main pkg inside the `cmd/appd` package. But I changed it to `.` instead `./...`. Its more clear

Reply:
and I also created the new build method and reverted the `InstallAll` method

Reply:
done on [`9249e08`](https://github.com/tendermint/starport/pull/1521/commits/9249e087ddf40d1097030ff6ffd063fa3725ead7) and [`fa892bc`](https://github.com/tendermint/starport/pull/1521/commits/fa892bc3e4adfc45c5e7cf7d590a8c37ac8ddf41)